packages/modular-scripts/src/cli.ts

```diff
@@ -125,6 +125,10 @@ program
     testOptions.watchAll.description,
     !process.env.CI,
   )
+  .option('--bail', testOptions.bail.description)
+  .option('--clearCache', testOptions.clearCache.description)
```

Comment:
Same with this - default to true in CI.

Reply:
We can't default this one to true because it exits the script once it's cleared the cache. It can't be combined with the actual test command or it will never run

Reply:
Ah - fair enough then! Maybe it's ```--no-cache``` we want to default to true on CI then.